From a4d2d8170ae055786a496fdd0baba4f2b8a8d71a Mon Sep 17 00:00:00 2001
From: Benjamin Drung <benjamin.drung@profitbricks.com>
Date: Thu, 24 Aug 2017 16:39:15 +0200
Subject: [PATCH] Support setting ENABLE_LUA_SUPPORT

To allow users to disable Lua support, do not overwrite the
ENABLE_LUA_SUPPORT variable. Instead just set the default if it is not
set yet.
---
 src/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index d87abc0..6f3a10e 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -214,7 +214,7 @@ endif()
 # We do not enable it by default, it's testing feature
 # If you want it please build with:
 # cmake -DENABLE_LUA_SUPPORT=ON .. 
-set(ENABLE_LUA_SUPPORT yes)
+set(ENABLE_LUA_SUPPORT yes CACHE BOOL "Enable Lua support")
 if (ENABLE_LUA_SUPPORT)
     message(STATUS "We will enable LuaJIT support")
 
-- 
2.11.0

